.github/workflows/deploy.yml

```
@@ -10,47 +10,47 @@ jobs:
-            git clone --depth=1 git@github.com:frankbhome/climate-compare.git /var/www/html
-          else
-            cd /var/www/html
-            git fetch origin main
-            git reset --hard origin/main
-          fi
\ No newline at end of file
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)